R/time_bins.R

```diff
@@ -377,12 +438,12 @@ time_bins <- function(interval = "Phanerozoic", rank = "stage", size = NULL,
         col = df$colour[i]
       )
     }
-    if (is.numeric(size)) {
+    if (is.numeric(size) || is.numeric(n_bins)) {
       title(paste(
```

Comment:
I think we should use paste0 here, otherwise we get a space before the last parenthesis

Reply:
Implemented